justfile.fedimint.just

```diff
@@ -23,7 +23,7 @@ test-count:
   ./scripts/tests/test-cov.sh
 
 test-compatibility *VERSIONS:
-  ./scripts/tests/backwards-compatibility-test.sh {{VERSIONS}}
+  ./scripts/tests/test-ci-all.sh {{VERSIONS}}
```

Comment:
For those using `just test-compatibility`, this change removes the default so you now need to explicitly pass which versions to test (e.g. `just test-compatibility v0.2.1`)